components/NodeBox/NodeBox.tsx

```diff
@@ -41,9 +42,6 @@ export default NodeBox
 const nodeBoxStyles = css`
   div {
     display: flex;
-  }
-  div > :global(.pointer-box)  {
-    transform: translateX(42%);
-    margin-right:5vw;
+    margin-right: -30%;
```

Comment:
it was a quickfix, we shouldn't do it this way,
on the call we can take a look and discuss it